### PR TITLE
Update build instructions

### LINKED ISSUE #630: [security issue] Sentry Auth token exposed in eas.json
**Describe the bug**

89c975413cd543fbb683b11bec984fc2163d9a77312c41c0b4480a570f3daa65

According to this thread:

https://github.com/expo/sentry-expo/issues/321 

you don't want to expose the auth token and use env variables instead. The thread is about using it in app.json but I think it applies here as well since the the security vulnerability is in others knowing the token first and foremost.

**To Reproduce**

1. Check eas.json
2. Confirm token is there

**Expected behavior**

Use env variables that are loaded via dot env where needed and not committed to the repo.

### PR DESCRIPTION
**Pull request by ansh**
Fixes #630

### CODE REVIEW

File: docs/build.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
   - iOS: `yarn ios`
   - Android: `yarn android`
   - Web: `yarn web`
+- If you are cloning or forking this repo as an open source developer, please check the intrsuctions below as well
```
Comment (hs4man21) [inline docs/build.md:21]: Small typo here: `instructions`

Comment (ansh) [reply docs/build.md:21]: Thanks

**Comment by ansh** [comment]
This should be squashed and merged when it is merged.

File: docs/build.md [hunk 2]
```diff
@@ -18,13 +18,16 @@
...
   - Web: `yarn web`
+- If you are cloning or forking this repo as an open source developer, please check the tips below as well
 - Run e2e tests
   - Start in various console tabs:
     - `yarn e2e:server`
     - `yarn e2e:metro`
   - Run once: `yarn e2e:build`
   - Each test run: `yarn e2e:run`
 - Tips
+  - Make sure you clone the `.env` and add the appropiate tokens (e.g. `SENTRY_AUTH_TOKEN` can be created on the Sentry dashboard). If this is not required, you can remove it from `eas.json` and `package.json`, as well as any mentions in the code.
```
Comment (jasikpark) [inline docs/build.md:29]:
```suggestion
  - Make sure you copy the `.env.example` to `.env` and add the appropiate tokens (e.g. `SENTRY_AUTH_TOKEN` can be created on the Sentry dashboard). If this is not required, you can remove it from `eas.json` and `package.json`, as well as any mentions in the code.
```


maybe more clear?

**Comment by Blainegunn** [comment]
any chance I can get an invite?

File: docs/build.md [hunk 3]
```diff
@@ -18,13 +18,16 @@
...
   - Web: `yarn web`
+- If you are cloning or forking this repo as an open source developer, please check the tips below as well
 - Run e2e tests
   - Start in various console tabs:
     - `yarn e2e:server`
     - `yarn e2e:metro`
   - Run once: `yarn e2e:build`
   - Each test run: `yarn e2e:run`
 - Tips
+  - Make sure you copy the `.env.example` to `.env` and add the appropiate tokens (e.g. `SENTRY_AUTH_TOKEN` can be created on the Sentry dashboard). If this is not required, you can remove it from `eas.json` and `package.json`, as well as any mentions in the code.
```
Comment (llllvvuu) [inline docs/build.md:29]: this doesn't fix for me, here is what I had to do: https://github.com/bluesky-social/social-app/issues/661

**Review by pfrazee — APPROVED** [review]
👍 LGTM. For future reference, people should refer to the expo docs to see which scopes their auth token requires.

**Pull request merged**